client/src/app/primitives/modal/Modal.less

```diff
@@ -31,4 +31,17 @@
   border-radius: 2px;
   padding: 20px;
   z-index: 1001;
-}
\ No newline at end of file
+
+  .close {
+    vertical-align: middle;
+    font-size: 2em;
```

Comment:
Do we actually want the X button to have a variable size? For the deployment modal it's now 26.4px while for the keyboard shortcuts it's 18px.

Reply:
Made it `rem` so the position and size is always fixed.